docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/OwnerUndefined.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Drupal\va_gov_backend\Plugin\Validation\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * Checks that the owner isn't set to none.
+ *
+ * @Constraint(
+ *   id = "OwnerUndefined",
+ *   label = @Translation("Owner Undefined", context = "Validation"),
+ *   type = "string"
+ * )
+ */
+class OwnerUndefined extends Constraint {
```

Comment:
```suggestion
class FieldAdministrationUndefined extends Constraint {
```

The filename needs to be renamed to match.